src/components/checkbox/TodoCheckbox.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import Checkbox from '@material-ui/core/Checkbox';
+
+interface Props {
+  itemId: number;
+  chosenItems: number[];
+  setChosenItems: (items: number[]) => void;
+  isDisabled?: boolean;
+}
+
+const TodoCheckbox = ({ itemId, chosenItems, setChosenItems, isDisabled }: Props) => {
```

Comment:
I prefer this syntax `export const TodoCheckbox: React.FC<Props> = ({ itemId, chosenItems, setChosenItems, isDisabled }) => {`

Reply:
:+1

Reply:
But I need to `export default` this component as Armen said in the first comment.

Reply:
User-in Listumy ogtagortums `TodoCheckbox` anunen shpotechnuma.
Prosto poxe **Checkbox** tir, qani ver inqu menak Todo-in chi patkanum.

Ev nuin dzevi ham UserListumy ham TodoListumy poxe arden.
```
import { Checkbox } from '../../checkbox/Checkbox';
import { Pagination } from '../../pagination/Pagination';
```

Reply:
**TodoCheckbox** componenty stexcelis ogtvalum `react-material`-i `Checkbox` componentic. Tra heti im komponenti anuny karumchum Checkbox tinim. Arden ti anun goyutyun one